### ISSUE TITLE
Documentation error for title of page cookbooks-partitioned

### ISSUE THREAD

**Issue by jiazhai** [apache/pulsar#3349]
**Describe the bug**
Documentation error for tile of page: http://pulsar.apache.org/docs/en/cookbooks-partitioned/

**To Reproduce**
open link: 
http://pulsar.apache.org/docs/en/cookbooks-partitioned/

**Expected behavior**
tile and content match.

**Screenshots**
![image](https://user-images.githubusercontent.com/6348208/50961653-3b91b200-1503-11e9-86db-206706ce809d.png)

**Comment by jiazhai** [comment]
This should be an easy fix in cookbooks-partitioned.md
```
---
id: cookbooks-partitioned
title: Non-persistent messaging  < === 
sidebar_label: Partitioned Topics
---
```

Need to check if other versions also have issue.

**Comment by sijie** [comment]
This is fixed by #3353
